spark_ec2.py

```diff
@@ -162,11 +183,74 @@ def setup_external_libs(libs):
 from boto.ec2.blockdevicemapping import BlockDeviceMapping, BlockDeviceType, EBSBlockDeviceType
 from boto import ec2
 
+import yaml
 
 class UsageError(Exception):
     pass
 
 
+def process_conf_file(file_path, parser):
```

Comment:
It will also be good to test cases where say the mapping fails (i.e. say invalid config option name) or the mapping has multiple values etc.